docs/_layouts/docs.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE>
+<html lang="en-us">
+{% include head.html %}
+<body class="docs">
+    {% include navigation-dropdown.html %}
+    <header class="header">
+        <div class="container">
+            <div class="row">
+              <div class="col-md-8">
+                <div class="title-container">
+                    <h1 class="display-2 text-white font-condensed mb-4">{{ page.title }}</h1>
+                    {% if page.subtitle %}
+                        <span class="subtitle text-white heading-underline" style="text-transform: none;">{{ page.subtitle }}</span>
```

Comment:
Just adding this for reference: I've added a task to add a `.text-transform-none` for the next alpha release of the framework so that we don't have to use inline styles for unsetting transforms like this.